doc/manual/getting_started_mgmt.rst

```diff
@@ -58,9 +58,15 @@ In other words, a worker created by the master has executed the experiment and c
 
 .. tip::
 
+    Before connecting master and client, please make sure your pc's and network's firewall do not block traffic on the ports used. Or you may try using ssh port forwarding.
+    
     In order to run the master and the clients on different PCs, start the master with a ``--bind`` flag: ::
 
-        $ artiq_master --bind [hostname or IP to bind to]
+        $ artiq_master --bind [master computer's hostname or IP to bind to]
+
+    use "\\\*", escaped \*, to indicate bind to all interfaces, for example: ::
```

Comment:
It's actually just ``*``, the ``\`` is simply required on most shells to disable glob expansion. Please be accurate.

Reply:
Also that's the situation on UNIX, but does the Windows shell work similarly?